### ISSUE TITLE
Disable autoplay on MetaCritic

### ISSUE THREAD

**Issue by justinmayer** [AdguardTeam/Userscripts#9]
The disableAutoplay userscript does not currently prevent autoplaying videos on MetaCritic.

Test URL: http://www.metacritic.com/movie/la-la-land

**Comment by ameshkov** [comment]
Sorry for the late reply. The issue is resolved now.

**Comment by justinmayer** [comment]
Thanks for your work on this, Andrey. I have tested the latest disableAutoplay userscript version 1.0.3 with the test URL above on two different computers, and it does indeed seem that auto-play is now off by default. A welcome change. Thank you!

As an ancillary comment, I noticed that when in Safari's "incognito mode", the video on that page still auto-plays on both machines. Test environment:

* macOS 10.12.3
* Safari 10.0.3
* TamperMonkey extension 4.2.5349
* Disable Video Autoplay userscript 1.0.3

I thought I would mention that in case it's something you would like to investigate further. For the moment, I'm content to avoid incognito mode for this particular site.
